command/title_body_survey.go

```diff
@@ -352,6 +360,8 @@ func titleBodySurvey(cmd *cobra.Command, issueState *issueMetadataState, apiClie
 		}
 	}
 
+	issueState.Labels = append(issueState.Labels, templateLabels...)
```

Comment:
Hi, thank you for attempting to solve this, but this solution is not going to work for anyone who fills in a template but does not have write or triage access to a repository. Most folks who open issues in open source repositories will  usually _not_ have privileges to assign arbitrary labels.

However, while filling out issues via web UI, some labels might be inherited from the template that was used. We definitely want this functionality in CLI as well, but for that we would need to pass the template name to the issue create GraphQL mutation. Currently, the API for that is _private_ to us, so we can not ask you to try to use that in this PR. For now I will close this and someone from our team is going to pick up the original issue soon and work on making this happen.

Thanks again!